tests/framework/ab_test.py

```diff
@@ -113,8 +113,8 @@ def binary_ab_test(
     Similar to `git_ab_test`, but instead of locally checking out different revisions, it operates on
     directories containing firecracker/jailer binaries
     """
-    result_a = test_runner(a_directory, True)
-    result_b = test_runner(b_directory, False)
+    result_a = test_runner("A", a_directory, True)
+    result_b = test_runner("B", b_directory, False)
```

Comment:
This additional parameter shouldn't be needed, since we already have the boolean

Reply:
fixef